web/static/js/reducers/idea.js

```diff
@@ -1,5 +1,7 @@
 const idea = (state = [], action) => {
   switch (action.type) {
+    case "SET_IDEAS":
```

Comment:
@vanderhoop If the idea is to fetch all of the ideas, I think this looks good. Would this be the same as ```return action.ideas``` ?